pkg/server/server_authz_test.go

```diff
@@ -567,6 +571,54 @@ func TestWrite(t *testing.T) {
 			})
 			require.NoError(t, err)
 		})
+
+		t.Run("errors_when_sending_more_than_max_modules", func(t *testing.T) {
+			authz.MaxModulesInRequest = 1
```

Comment:
Could we instead make `MaxModulesInRequest` a constant and tweak the test behaviors based on the value. For example here, setup the tests so that it sends `authz.MaxModulesInRequest+1` modules.